expr/range.go

```diff
@@ -67,10 +77,23 @@ func (e *Range) unmarshal(data []byte) error {
 		case unix.NFTA_RANGE_SREG:
 			e.Register = ad.Uint32()
 		case unix.NFTA_RANGE_FROM_DATA:
-			e.FromData = ad.Uint32()
+			e.FromData = ad.Bytes()
 		case unix.NFTA_RANGE_TO_DATA:
-			e.ToData = ad.Uint32()
+			e.ToData = ad.Bytes()
 		}
 	}
 	return ad.Err()
 }
+
+func nestedAttr(data []byte, attrType uint16) ([]byte, error) {
+	ae := netlink.NewAttributeEncoder()
+	ae.Do(unix.NLA_F_NESTED|attrType, func() ([]byte, error) {
+		nae := netlink.NewAttributeEncoder()
+		// You can set endianness if you need to for the data.
```

Comment:
This comment looks like a left-over from debugging? Can you remove it?

Reply:
done